examples/demo-rollup/README.md

```diff
@@ -229,88 +225,76 @@ Here's an example of a JSON representing the above call:
+Commands:
+  bank                Generates a transaction for the `bank` module
+  sequencer-registry  Generates a transaction for the `sequencer_registry` module
+  election            Generates a transaction for the `election` module
+  value-setter        Generates a transaction for the `value_setter` module
+  accounts            Generates a transaction for the `accounts` module
+  help                Print this message or the help of the given subcommand(s)
 
-```bash
-$ cat ./examples/test-data/requests/transfer.dat
-5ef848746e8d2b9c27ee46210e185dc9f3b690d5cef42a13fb9c336bd40c798210bf7af613997f7af57c9681a242f5fe4121a1539ba4f5f32f14c49f978b990a7b758bf2e7670fafaf6bf0015ce0ff5aa802306fc7e3f45762853ffc37180fe64a0000000001fea6ac5b8751120fb62fff67b54d2eac66aef307c7dde1d394dea1e09e43dd44c800000000000000135d23aee8cb15c890831ff36db170157acaac31df9bba6cd40e7329e608eabd0000000000000000
+Options:
+  -h, --help  Print help
 ```
 
-The above is the hex representation of the serialized transaction.
-
-#### 3. Bundle the Serialized Transaction
-
-After serializing your transactions (just one in this case), you must bundle them into a blob. You can use the `sov-cli make-batch` subcommand:
+Let's go ahead and import the transaction into the wallet
 
 ```bash
-$ ./target/debug/sov-cli make-batch -h
-Usage: sov-cli make-batch [PATH_LIST]...
-
-Arguments:
-  [PATH_LIST]...  List of serialized transactions
+$ ./target/debug/sov-cli transactions import from-file bank --path ./examples/test-data/requests/transfer.json
+Adding the following transaction to batch:
+{
+  "bank": {
+    "Transfer": {
+      "to": "sov1l6n2cku82yfqld30lanm2nfw43n2auc8clw7r5u5m6s7p8jrm4zqrr8r94",
+      "coins": {
+        "amount": 200,
+        "token_address": "sov16m8fxq0x5wc5aw75fx9rus2p7g2l22zf4re72c3m058g77cdjemsavg2ft"
+      }
+    }
+  }
+}
 ```
 
-Use the command below to store the serialized blob in `./examples/test-data/requests/tx_blob`:
-
-```bash
-$ ./target/debug/sov-cli make-batch ./examples/test-data/requests/transfer.dat > ./examples/test-data/requests/tx_blob
-$ cat ./examples/test-data/requests/tx_blob
-01000000b60000005ef848746e8d2b9c27ee46210e185dc9f3b690d5cef42a13fb9c336bd40c798210bf7af613997f7af57c9681a242f5fe4121a1539ba4f5f32f14c49f978b990a7b758bf2e7670fafaf6bf0015ce0ff5aa802306fc7e3f45762853ffc37180fe64a0000000001fea6ac5b8751120fb62fff67b54d2eac66aef307c7dde1d394dea1e09e43dd44c800000000000000135d23aee8cb15c890831ff36db170157acaac31df9bba6cd40e7329e608eabd0000000000000000
-```
+This output indicates that the wallet has saved the transaction details for later signing.
 
-#### 4. Submit the Transaction
+#### 3. Submit the Transaction(s)
 
-You now have a blob with one serialized transaction in `./examples/test-data/requests/tx_blob`. Switch back to the `examples/demo-rollup` directory and use the Makefile to submit it:
+You now have a batch with a single transaction in your wallet. If you want to submit any more transactions as part of this
```

Comment:
This PR is great. 

Just a small nit, not sure, but wondering if  the usage of word "wallet" in this tutorial might be confusing to folks. 

Because previously we refer to the same tool as `sov-cli` everywhere. Thoughts @preston-evans98?